examples/cleanup-controller/deployment.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: apps/v1
```

Comment:
Can you put these files here instead? https://github.com/kubernetes-sigs/sig-storage-local-static-provisioner/tree/master/deployment/kubernetes/example

Can create a new directory for this

Reply:
Done